components/Badge/index.tsx

```diff
@@ -1,14 +1,28 @@
 import Link from "next/link";
+import { ReactEventHandler } from "react";
+
+interface BadgeProps {
+  name: string;
+  id: string | number;
+  avatar: string;
+  tokens: string | number;
+  owned: boolean;
+}
+
+export default function Badge({ name, id, avatar, tokens, owned }: BadgeProps) {
+  const imageOnError: ReactEventHandler<HTMLImageElement> = (e) => {
+    e.currentTarget.src = "/images/badges/badge-not-found.svg";
+  };
 
-export default function Badge({ name, id, avatar, tokens, owned }) {
   return (
     <Link
       href={`/badge/${id}`}
       className={`h-full w-full ${owned ? "opacity-100" : "opacity-30"}`}
     >
       <div>
-        <img src={avatar} alt={name}></img>
+        <img src={avatar} onError={imageOnError} alt="Badge Image"></img>
```

Comment:
```suggestion
        <img src={avatar} onError={imageOnError} alt={name}></img>
```
For accessibility reasons.